test/integration/smoke/test_list_ids_parameter.py

```diff
@@ -194,10 +194,7 @@ def setUpClass(cls):
         cls._cleanup = [
                         cls.disk_offering,
                         cls.account,
-                        cls.service_offering,
-                        cls.snapshot_1,
-                        cls.snapshot_2,
-                        cls.snapshot_3
+                        cls.service_offering
```

Comment:
@swill I wanted the out-of-band management PR to go green so it could be merged soon without missing the freeze date, that's why I fixed it in my PR.